pkg/render/render.go

```diff
@@ -19,8 +19,8 @@ import (
 	"bytes"
 	"context"
 	"fmt"
+	"html/template"
```

Comment:
Fun bug

Reply:
It would be a pain, but switch to https://pkg.go.dev/github.com/google/safehtml?tab=doc ?

Reply:
Hmm - that seems to be more of an HTML builder-ing library.

Reply:
So it's the same API as html/template, but it's to prevent XSS (https://static.googleusercontent.com/media/research.google.com/en//pubs/archive/42934.pdf has a bunch of examples). https://docs.google.com/document/d/1oNlv8Itpr_J59JNKJ0Jd4U93MJmZvXHrB0p8izQdGsM/edit also gives a bunch of examples on security vulns it protects against. 

I think it's probably too much work for now, but might be worth a TODO.